_data/meltano/utilities/superset/apache.yml

```
@@ -131,3 +131,11 @@ next_steps: |
      ```
 
   Now when you (re)start Superset, you will be able to connect to a new type of database, like Snowflake in the example.
+usage: |
+  ## Troubleshooting
+  
+  ### Secret Key Change
+  
+  Be aware that if you change your secret key, the metadata database will become unreadable. If this happens, you can delete it at `$MELTANO_PROJECT_ROOT/.meltano/utilities/superset/superset.db`. 
```

````suggestion
  If you change your [`SECRET_KEY`](#secret-key), the metadata database containing your users and reports will become unreadable. If there was nothing valuable in there, you can delete it at `.meltano/utilities/superset/superset.db` inside your project. If you changed the secret key because the old one was leaked, you can [rotate the key](https://superset.apache.org/docs/installation/configuring-superset/#secret_key-rotation):
  
  ```bash
  meltano config superset set SECRET_KEY <new_key>
  meltano config superset set PREVIOUS_SECRET_KEY <original_key>
  meltano invoke superset re-encrypt-secrets
  ```
````